src/igvfd/schemas/crispr_modification.json

```diff
@@ -127,11 +127,17 @@
                 "igvf_utils": "activation"
             }
         },
-        "tagged_protein": {
-            "title": "Tagged Protein",
-            "description": "The tagged protein in modifications in which the Cas nuclease is fused to an antibody.",
-            "type": "string",
-            "linkTo": "Gene",
+        "tagged_proteins": {
+            "title": "Tagged Proteins",
+            "description": "The tagged proteins in which the Cas nuclease is fused to an antibody.",
+            "type": "array",
+            "minItems": 1,
```

Comment:
Should this also have a maxItems? Since it was originally a string and could only have 1 item.

Reply:
I'm not sure what you are suggesting by this request, do you think there will only ever be one tagged protein for CRISPR modifications?